data/persons.txt

```diff
@@ -611,3 +611,161 @@ ship "Modified Osprey"
 	explode "small explosion" 40
 	explode "tiny explosion" 28
 	"final explode" "final explosion large" 1
+
+
+person "Local God"
+	government "Author"
+	frequency 200
+	personality
+		mining harvests surveillance unconstrained forebearing
+	phrase
+		phrase
+			"friendly author"
+	phrase
+		word
+			"Search for Ursa Polaris Station!"
+	phrase
+		word
+			"Someday"
+			"One day"
+			"Eventually"
+		word
+			", "
+		word
+			"maybe I should "
+			"I might "
+			"I will "
+			"I'm going to "
+		word
+			"park this heap"
+			"moor this beast"
+			"put down roots"
+			"find a cozy orbit"
+		word
+			" and "
+		word
+			"retire as an outfit dealer."
+			"open a univeral outfitter."
+			"start dealing exotic outfits."
+			"sell outfits from everywhere."
+			"stock outfits from all over the place."
+	phrase
+		word
+			"If only "
+			"I wish "
+			"Reputable Captains wish "
+		word
+			"someone "
+			"a rich Captain "
+			"an entrepreneurial spirited individual "
+			"a Captain with wealthy friends "
+		word
+			"would "
+		word
+			"build "
+			"construct "
+			"put together "
+			"cobble together "
+			"make "
+		word
+			"and stock "
+		word
+			"a universal outfitter space station."
+			"a place you could buy any outfit."
+			"somewhere you could optimize your ships, without cris-crossing the known galaxy."
+			"a space station out of asteroids - that carried all the outfits."
+	phrase
+		word
+			"I "
+		word
+			"stole "
+			"captured "
+			"restored "
+			"found "
+			"got a hold of "
+			"appropriated "
+			"procured "
+ 		word
+			"this very old Korath World Ship and "
+		word
+			"made it my own"
+			"customized it"
+			"made some modifications"
+			"made a few modifications"
+		word
+			", with a little help, of course."
+	ship "Ursa Polaris" "Local God"
+		"never disabled"
+
+ship "Ursa Polaris"
+	sprite "ship/localworldship"
+	attributes
+		category "Heavy Warship"
+		cost 142857
+		shields 39000
+		hull 27000
+		"required crew" 559
+		"bunks" 972
+		"mass" 1839
+		"drag" 12
+		"heat dissipation" .9
+		
```

Comment:
Looks like you have some extra whitespace here.

Reply:
from where i removed the cloak attribute...lol